internal/pkg/service/stream/api/mapper/sink_response.go

```diff
@@ -84,6 +84,28 @@ func (m *Mapper) newTableSinkResponse(entity *definition.TableSink) (out api.Tab
 	return out, nil
 }
 
+func (m *Mapper) NewSinkVersions(
+	ctx context.Context,
+	afterId string,
+	limit int,
+	list func(...iterator.Option) iterator.DefinitionT[definition.Sink],
+) (*api.EntityVersions, error) {
+	sources, page, err := loadPage(ctx, afterId, limit, etcd.SortAscend, list, m.NewSinkResponse)
+	if err != nil {
+		return nil, err
+	}
+
+	versions := make([]*api.Version, 0)
```

Comment:
We could do a minor optimization by passing `len(sink.Version)` instead of 0 but up to you, it's not necessary. Same for sources.

Reply:
There is linter for that. We can merge it when rebased :)